### DISCUSSION TITLE
Where is the transfered file saved on IPhone / how to access the file?

### DISCUSSION THREAD

**Discussion by bjce** [localsend/localsend#1410] [Discussion (English, International)]
Hello everyone

First of all thanks a lot to the developpers/maintainers for this great piece of software. I could succesfully transfer a text file from a linux machine to my IPhone however I cannot locate it on my device. Where has it been saved and how can I access it?
I hope there were no similar questions already answered but at least I could not find such. Many thanks in advance for your support.

**Comment by Tienisto** [comment]
You can find it in the "Files" app of the iPhone. Alternatively, you can view the history in the "Receive" page

**Reply by PorkBellyNZ** [reply]
I have the same issue that happened on my iPad. I tried to find the file from the history list and located its address. It is under the VAR file and I am unable to access it. At present, I can only use it by copying it to other folders. But the files in the download folder cannot be moved or deleted. Is there any way to change the default download folder location? Thank you very much for developing this software. !!!

**Reply by MiranoVerhoef** [reply]
> You can find it in the "Files" app of the iPhone. Alternatively, you can view the history in the "Receive" page

Hey Maintainer,

Excuses for opening or responding to a old issue. Reason for it being that its a first hit on google, hence the response. I've just did a ISO move from my Windows PC to my IOS device, yet the ISO is located at: /var/mobile/containers/data/application/****/Documents/***.iso

This gives a few problems:
1. Not being able to delete the file in question
2. Not able to go to the location through the "Files" APP
3. The "Show in Folder" also doesn't function as expected
4. Also no way to change default store location on IOS

Is it possible to get a fix on this? :)

**Comment by bjce** [comment]
@Tienisto Thanks for your answer. And thanks again for the great piece of software

**Comment by badjano** [comment]
similar issue https://github.com/localsend/localsend/issues/2911
